wrapper:
  config_file_name: avia.yaml
  lidar_topic: /livox/lidar_192_168_1_159
  imu_topic: /livox/imu_192_168_1_159
  lidar_type: 0 # 0)avia
  blind: 1

front_end:
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  # extrin_t: [0.4165,0.2326,-0.2840]  # 西大数据集
  extrin_t: [-0.011,-0.02329,0.04412] # mid360
  filter_leaf_size: 0.2
ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001
map:
  map_side_length_2: 500.0
  map_resolution: 0.5